pizza-time/src/components/card/card.js

```diff
@@ -15,7 +15,9 @@ class Card extends React.Component {
       comments: [],
       showMessages: false,
       attendees: [],
-      showActions : this.props.showActions
+      showActions : this.props.showActions,
+      host : "localhost:5500"
```

Comment:
switch this to use the deployed backend instead of localhost, test that it is still working then re-PR. I am going to deploy tonight to Master so everything needs to be running from backend.